.gitignore

```diff
@@ -348,3 +348,7 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+*/.DS_Store
+*/node_modules
```

Comment:
node_modules are already on line 276. Can you add this

 ## MAC files ##
.DS_Store

## NPM ##
npm
npm-debug.log